sphinx/writers/latex.py

```diff
@@ -1240,7 +1240,6 @@ def visit_image(self, node: Element) -> None:
                 (1, 'middle'): ('\\raisebox{-0.5\\height}{', '}'),
                 (1, 'bottom'): ('\\raisebox{-\\height}{', '}'),
                 (0, 'center'): ('{\\hspace*{\\fill}', '\\hspace*{\\fill}}'),
-                (0, 'default'): ('{\\hspace*{\\fill}', '\\hspace*{\\fill}}'),
```

Comment:
I think this might be changing the current behavior. Need to set `attrs.setdefault('align', 'default')` and keep this line?

Reply:
I thought nobody sets `'default'` to the `align` attribute of the image nodes (Our transform component only sets it to the figure and table nodes!). It means this code would be never used.

Reply:
I see. So this line wasn't ever used, even before this patch?

Reply:
Yes, I think so.